pages/common/docker-save.md

```diff
@@ -0,0 +1,24 @@
+# docker save
+
+> Export one or more docker images to archive.
+> More information: <https://docs.docker.com/engine/reference/commandline/save/>.
+
+- Save the image by redirecting output to a file:
+
+`docker save {{image}} > {{path/to/file}}`
+
+- Save an image to a tar archive:
+
+`docker save --output {{/path/to/file.tar}} {{image}}`
+
+- Save all tags of the image:
+
+`docker save --output {{/path/to/file.tar}} {{image_name}}`
+
+- Use gzip to save the image:
```

Comment:
Generally, we avoid documenting the functionality of other commands on a given page.